model/postgres/record.py

```diff
@@ -57,3 +59,20 @@ def __dir__(self):
     def __iter__(self):
         for attr in dir(self):
             yield attr, getattr(self, attr)
+
+    @hybrid_property
+    def has_version(self):
+        return self._has_version
+
+    @hybrid_property
+    def getLanguage(self):
+        return self.languages[0]
```

Comment:
a minor nit, if we aren't using this method we should remove it!

Reply:
Sure thing.